testing/matrix-sdk-integration-testing/Cargo.toml

```diff
@@ -16,7 +16,8 @@ futures = { version = "0.3.29", features = ["executor"] }
 futures-core = { workspace = true }
 futures-util = { workspace = true }
 http = { workspace = true }
-matrix-sdk = { workspace = true, default-features = true, features = ["testing", "qrcode"] }
+matrix-sdk-base = { workspace = true, default-features = true, features = ["testing", "qrcode"] }
```

Comment:
Might be nice to preserve the sort order, base comes after the one without a suffix.

Reply:
Done, thanks